providers/fcontext.rb

```diff
@@ -38,8 +38,10 @@ def semanage_options(file_type)
 
 # Run restorecon to fix label
 action :relabel do
-  res = shell_out!('restorecon', '-irv', new_resource.file_spec)
-  new_resource.updated_by_last_action(true) unless res.stdout.empty?
+  execute "selinux-fcontext-#{new_resource.secontext}-relabel" do
+    command "find / -regextype posix-egrep -regex '#{new_resource.file_spec}' -type d -print0  2>/dev/null | xargs -0 restorecon -iRv"
```

Comment:
This worries me because of shell escaping interfering with the regex. We might need to escape it. I think the best way to handle this is using `shell_out` as it'll avoid shell expansion.
Also, `find BLABABA -exec restorecon -iRv {} \;` is better IMO